py-pkg/Makefile

```diff
@@ -0,0 +1,20 @@
+
+install:
+	# Install uptasticsearch Python package
+	cd py-pkg && \
```

Comment:
can you add one more issue around moving R into `r-pkg/` for consistency? That would include updating any documentation, Travis, etc. so definitely should be its own PR

Reply:
#87